spec/test_app/app/controllers/basechurch/v1/announcements_controller_spec.rb

```diff
@@ -24,6 +24,7 @@
       expect(body['announcements']['id']).to eq(announcement.id.to_s)
       expect(body['announcements']['description']).to eq(announcement.description)
       expect(body['announcements']['position']).to eq(announcement.position)
+      expect(body['announcements']['url']).to eq(announcement.url)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.